src/osip-monitoring-controller/src/main/java/edu/kit/pse/osip/monitoring/controller/MonitoringController.java

```diff
@@ -111,7 +111,7 @@ public void start(Stage primaryStage) {
             try {
                 setupClients();
             } catch (UAClientException e) {
-                System.err.println("Unable to connect to servers. " + e.getMessage());
+                System.err.println("Unable to connect to servers. " + e.getLocalizedMessage());
```

Comment:
`getMessage` sollte, soweit ich weiß, eine englische Ausgabe haben, und `getLocalizedMessage` eine Ausgabe in der eingestellten Sprache. Wenn man OSIP übersetzen will, dann macht es Sinn `getLocalizedMessage` zu verwenden. Mit dieser Änderung ist das einfacher.

Ich bin kein Freund von Stack Traces auf Sysout, weil sie mMn. schlecht für die Userexperience sind.